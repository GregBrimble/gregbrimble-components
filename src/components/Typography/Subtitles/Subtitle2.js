import styled from "styled-components"
import { Subtitle } from "./shared"

const Subtitle2 = styled(Subtitle)`
  font-size: 0.875rem;
  line-height: 1.375rem;
  font-weight: 600;
`

export default Subtitle2
