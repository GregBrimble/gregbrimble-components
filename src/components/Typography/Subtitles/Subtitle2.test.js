import React from 'react';
import Subtitle2 from './Subtitle2';
import renderer from 'react-test-renderer';

describe('Subtitle2', () => {
    it('renders properly', () => {
        const tree = renderer
            .create(<Subtitle2>Subtitle 2</Subtitle2>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
