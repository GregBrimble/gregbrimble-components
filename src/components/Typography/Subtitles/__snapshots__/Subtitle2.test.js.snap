// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle2 renders properly 1`] = `
<h6
  className="sc-bdVaJa sc-bwzfXH iqdCJl"
>
  Subtitle 2
</h6>
`;
